packages/debugger-extension/src/index.ts

```diff
@@ -1470,6 +1473,142 @@ const debugConsole: JupyterFrontEndPlugin<void> = {
   }
 };
 
+const debugMenu: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/debugger-extension:debug-menu',
+  description: 'Menu bar item for interacting with the debugger',
+  autoStart: true,
+  requires: [IDebugger, IMainMenu, ITranslator, INotebookTracker],
+  optional: [ISessionContextDialogs],
+  activate: async (
+    app: JupyterFrontEnd,
+    debug: IDebugger,
+    mainMenu: IMainMenu,
+    translator: ITranslator,
+    notebookTracker: INotebookTracker,
+    sessionDialogs_: ISessionContextDialogs | null
+  ) => {
+    const trans = translator.load('jupyterlab');
+    const sessionDialogs =
+      sessionDialogs_ ?? new SessionContextDialogs({ translator });
+
+    app.commands.addCommand(Debugger.CommandIDs.setVariablesFilterOptions, {
+      label: args => trans.__(args.label as string),
+      caption: args => trans.__(args.label as string),
```

Comment:
All those new commands should only be enabled if the debugger service is running

```suggestion
      caption: args => trans.__(args.label as string),
      isEnabled: () => service.isStarted,
```